Documentation/ApiOverview/MountPoints/Index.rst

```diff
@@ -0,0 +1,253 @@
+.. include:: /Includes.txt
+.. index:: Mount points
+.. _Mount-points:
+
+============
+Mount points
+============
+
+.. highlight:: text
+
+.. contents:: This page
+   :backlinks: top
+   :local:
+
+
+.. how to write *Words:*
+..    mount point, multisite, multi language support, page tree, subpage
+
+
+What are mount points?
+======================
+
+TYPO3 editors can reuse existing pages of the page tree and insert them again
+at another location without having to copy anything. To achieve this they
+create a new page and set its type to *MountPoint*. Then they select a page as
+content source. This is called *mounting*. Note that no pages nor content
+elements are copied. Instead, mounting creates a reference to content that is
+already maintained elsewhere in the installation. This also means that an
+update of that content will be shown in all mounted pages as well.
+
+Mounting may or may not include subpages, depending on what the editor chooses.
+It's an option. Various other properties of mounted pages can be specified as
+well in the properties of the mount point page. An important option of these is
+the one that determines what content should be shown on exactly the mount point
+page. This may be the referenced content or the content of mount point page.
+
+
+Why?
+====
+
+*Powerful and indespensible:* Mount points are an indespensible, powerful and
+important tool for huge enterprise installations and intranet or extranet
+solutions.
+
+*Efficiency:* Mount points allow editors to maintain content only once but
+reuse it in various sections of the website. Colors, style and appearance may
+vary depending on each rendering context.
+
+
+Terminology
+===========
+
+Mount point
+   *Mount point page* is an alias. This is a page (= page record) that has its
+   type set to *MountPoint*. This is done by setting field `doktype=7`.
+   Another, regular page of the page tree is selected as *content source*. This
+   is the "mounted page" or "mounted content". Specifying the mounted page is
+   called *mounting*, sometimes also labelled *web mount*. It is an option
+   whether **subpages** are included or not.
+
+
+Mounted page
+   An alias for this is *mount target* or *content source*. A mounted page is
+   an existing, regular page that serves as content source.
+
+
+Multisite
+   This is a setup of an TYPO3 installation where multiple sites, each with
+   their own domain and page tree, share the same installation.
+
+
+URLs
+====
+
+MP parameter
+------------
```

Comment:
I'm sorry to say this, but the rework in my eyes makes it worse. For example: the MP parameter will not appear as URL parameter anymore, therefore explaining anything about an "additional GET parameter" without directly explaining that there will be no GET parameter shown is kind of confusing.

Reply:
I now moved the MP stuff to a note about technical background, as it's basically "core dev only" information.